release.sh

```diff
@@ -1,8 +1,33 @@
-#!/usr/bin/env bash
+#!/bin/bash
+set -ex
 
+# ensure working dir is clean
+git status
+if [[ -z $(git status -s) ]]
+then
+  echo "tree is clean"
+else
+  echo "tree is dirty, please commit changes before running this"
+  exit 1
+fi
 
-git checkout -b v${FDK_VERSION}-stable
-git push origin v${FDK_VERSION}-stable
-PBR_VERSION=${FDK_VERSION} python setup.py sdist bdist_wheel
-twine upload dist/fdk-${FDK_VERSION}*
-git checkout master
+git pull
+
+version_file="fdk/version.py"
+current_version=$(grep -m1 -Eo "[0-9]+\.[0-9]+\.[0-9]+" ${version_file})
+new_version=$(docker run --rm marcelocorreia/semver semver -c -i patch ${current_version})
```

Comment:
there's treeder/bump too (used in CLI) - we should have an fnproject/bump since we're using it in lots of spots, not relying on 3rd party - this is ok for now, just pointing out

Reply:
Travis's image doesn't work for FDK python as it tries to calculate a version based on git history.
So, FYI, i tried and it didn't work because the version after bump was something like `0.0.12`. FDK already passed that version a long time ago.